src/main/java/java/lang/String.java

```diff
@@ -3988,12 +4042,11 @@ public static String format(String format, Object... args) {
      * @see  java.util.Formatter
      * @since  1.5
      *
-     * @diffblue.noSupport
+     * @diffblue.limitedSupport The locale argument is ignored
```

Comment:
:bulb: It's worth documenting that date/time related format specifiers are not supported. The example from the Javadoc of Formatter.java, `String.format("Duke's Birthday: %1$tm %1$te,%1$tY", c);`, gives an invariant violation.